fedimint-client/src/lib.rs

```diff
@@ -351,6 +507,19 @@ impl ClientInner {
 
         Ok(txid)
     }
+
+    async fn transaction_update_stream(
+        &self,
+        operation_id: OperationId,
+    ) -> BoxStream<OperationState<TxSubmissionStates>> {
+        self.executor
+            .notifier()
+            .module_notifier::<OperationState<TxSubmissionStates>>(
+                TRANSACTION_SUBMISSION_MODULE_INSTANCE,
```

Comment:
I haven't come across this yet. What does this represent?

Reply:
Since the executor only supports state machines in a module context I used `u16::MAX` as the fake modiule instance ID there. You can find the state machine responsible for submitting transactions here:

https://github.com/fedimint/fedimint/blob/33ccea92b85783e293c0903c9f7cab379d2b32f7/fedimint-client/src/transaction/sm.rs#L45-L84